gateway-service-livy/pom.xml

```diff
@@ -58,5 +65,10 @@
             <artifactId>gateway-test-utils</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
This shouldn't be necessary - Should be handled automatically for all submodules.